synth.py

```diff
@@ -69,6 +69,23 @@
 """
 )
 
+s.replace(
```

Comment:
is this something that should be templated at the synthtool level instead?

CC @busunkim96

Reply:
I think the microgenerator templates already behave like this. They don't raise if a version cannot be found. So this can probably be removed when the repo is transitioned. https://github.com/googleapis/gapic-generator-python/blob/af59c2c3c3d6b7e1f626c3fbc2c03f99ca31b4a4/gapic/templates/%25namespace/%25name_%25version/%25sub/services/%25service/client.py.j2#L715-L722

Reply:
ok, since that is still in progress - @tseaver mind adding a comment this can be removed later?

Reply:
@kolea2 Done in 2d703ef